src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExportClient/Grpc/GrpcStatusDeserializer.cs

```diff
@@ -85,7 +85,7 @@ internal static class GrpcStatusDeserializer
 
         foreach (var detail in status.Value.Details)
         {
-            if (detail.TypeUrl != null && detail.TypeUrl.EndsWith("/google.rpc.RetryInfo"))
+            if (detail.TypeUrl != null && detail.TypeUrl.EndsWith("/google.rpc.RetryInfo", StringComparison.Ordinal))
```

Comment:
All the files inside the Grpc folder are copied from the Grpc repo. We should not change their content unless it is updated in that repo.